.github/workflows/publish-bytecode.yaml

```
@@ -34,4 +34,4 @@ jobs:
       - uses: actions/upload-artifact@v2
         with:
           name: Bytecode-${{ steps.release_tag.outputs.VERSION }}
-          path: build/
+          path: artifacts/
```

```suggestion
          path: build/
```
to avoid breaking the deployment pipelines.

---

The underlying directory structure is different and has nested folders. This will be a breaking change anyways